stable/polaris/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: "0.3.0"
-appVersion: "0.2"
+version: "0.3.1"
```

Comment:
I'm still down to adjust the chart to release a 0.3.1 version, just without the LimitRange stuff if we can.